rabbitmq/check.py

```diff
@@ -308,10 +356,10 @@ def get_stats(self, instance, base_url, object_type, max_detailed, filters, cust
         if len(data) > max_detailed:
             # Display a warning in the info page
             self.warning(
-                "Too many queues to fetch. You must choose the %s you are interested in by editing the rabbitmq.yaml configuration file or get in touch with Datadog Support" % object_type)
+                "Too many items to fetch. You must choose the %s you are interested in by editing the rabbitmq.yaml configuration file or get in touch with Datadog Support" % object_type)
 
         for data_line in data[:max_detailed]:
-            # We truncate the list of nodes/queues if it's above the limit
+            # We truncate the list of nodes/queues/exchanges if it's above the limit
```

Comment:
Can you just change this to "We truncate the list if it's above the limit" or something like that?